### PR TITLE
Allow static headers for OTLP exporter

### PR DESCRIPTION
**Pull request by brendanmaguire**
* A new parameter is added which can be used to specify static headers to be included in the HTTP requests
* Defaults to an empty map

### CODE REVIEW

File: modules/opentelemetry-otlp-grpc-exporter/src/main/scala/io/janstenpickle/trace4cats/opentelemetry/otlp/OpenTelemetryOtlpGrpcSpanCompleter.scala [hunk 1]
```diff
@@ -14,10 +14,11 @@ object OpenTelemetryOtlpGrpcSpanCompleter {
     host: String = "localhost",
     port: Int = 4317,
     config: CompleterConfig = CompleterConfig(),
-    protocol: String = "http"
+    protocol: String = "http",
+    staticHeaders: Map[String, String] = Map.empty,
```
[the remark this answers is not in the document]
Comment (brendanmaguire) [reply modules/opentelemetry-otlp-grpc-exporter/src/main/scala/io/janstenpickle/trace4cats/opentelemetry/otlp/OpenTelemetryOtlpGrpcSpanCompleter.scala]: Yep, make sense. Change pushed.

**Comment by catostrophe** [comment]
@brendanmaguire Should we have added the same parameter to `io.janstenpickle.trace4cats.opentelemetry.otlp.OpenTelemetryOtlpHttpSpanExporter`?

**Comment by catostrophe** [comment]
@brendanmaguire I think it should've been `List[(CIString, String)]` since duplicate header names are allowed

**Comment by brendanmaguire** [comment]
Yep. Make sense to me.

**Pull request merged**